src/hooks/use-filtered-assets-list.hook.ts

```diff
@@ -1,11 +1,16 @@
-import { useMemo, useState } from 'react';
+import { useCallback, useMemo, useState } from 'react';
+import { useDispatch } from 'react-redux';
 
+import { setZeroBalancesShown } from '../store/wallet/wallet-actions';
+import { useHideZeroBalances } from '../store/wallet/wallet-selectors';
 import { TokenInterface } from '../token/interfaces/token.interface';
 import { isString } from '../utils/is-string';
 import { isNonZeroBalance } from '../utils/tezos.util';
 
-export const useFilteredAssetsList = (assetsList: TokenInterface[], initialIsHideZeroBalance = false) => {
-  const [isHideZeroBalance, setIsHideZeroBalance] = useState(initialIsHideZeroBalance);
+export const useFilteredAssetsList = (assetsList: TokenInterface[], overrideHide?: boolean) => {
+  const memoryState = useHideZeroBalances();
```

Comment:
use `useHideZeroBalancesSelector();` only where you need to pass this as argument